Codeinwp/neve
# Add integration for Formbricks survey in Dashboard

### Summary


- Added integration for Formbricks in Neve Dashboard.
- Added the `@formbricks/js` package and updated the `node-sass`.
- Updated JSDocs requested by js linter.

### Will affect visual aspect of the product

NO

### Screenshots 

<img width="2250" alt="Screenshot 2024-01-10 at 14 19 45" src="https://github.com/Codeinwp/neve/assets/17597852/704ad426-c4ef-4e32-b669-a4f7859c219e">

### Test instructions


When you enter in the Dashboard, after some seconds, you should see Formbricks widget in the bottom-right corner of the window.

## Check before Pull Request is ready:



Closes [#2723](https://github.com/Codeinwp/neve-pro-addon/issues/2723)
